pretalx_lower_thirds/forms.py

```diff
@@ -1,23 +1,21 @@
 from django import forms
 from django.utils.translation import gettext_lazy as _
 from hierarkey.forms import HierarkeyForm
+from i18nfield.forms import I18nFormField, I18nFormMixin, I18nTextInput
 
 
-class LowerThirdsSettingsForm(HierarkeyForm):
-    lower_thirds_no_talk_info = forms.CharField(
+class LowerThirdsSettingsForm(I18nFormMixin, HierarkeyForm):
+    lower_thirds_no_talk_info = I18nFormField(
         help_text=_(
-            "Will be shown as talk title if there's currently no talk "
-            "running."
+            "Will be shown as talk title if there's currently no talk " "running."
```

Comment:
That indentation was there to not have too long lines. Feel free to remove it, but please also remove the unneeded quotes then :)

Reply:
Sorry, I committed auto-formatter changes – I habitually run `black` before committing.